From 61f6148ec22f467d18d4dca58c3067cad1faabb8 Mon Sep 17 00:00:00 2001
From: Manolo van Ee <manolo.vanee@gmail.com>
Date: Thu, 11 Dec 2025 22:01:01 +0100
Subject: [PATCH 1/4] [stdlib Synchronization] Make _Cell publicly available

---
 stdlib/public/Synchronization/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/stdlib/public/Synchronization/CMakeLists.txt b/stdlib/public/Synchronization/CMakeLists.txt
index 299213ac577..422e7ef7fdc 100644
--- a/stdlib/public/Synchronization/CMakeLists.txt
+++ b/stdlib/public/Synchronization/CMakeLists.txt
@@ -199,6 +199,7 @@ if(SWIFT_SHOULD_BUILD_EMBEDDED_STDLIB)
       IS_FRAGILE
       
       ${SWIFT_SYNCHRONIZATION_ATOMIC_SOURCES}
+      Cell.swift
       GYB_SOURCES
         ${SWIFT_SYNCHRONIZATION_GYB_SOURCES}
 
-- 
2.53.0

